docs/workflows/lambda.md

```diff
@@ -6,123 +6,64 @@ hide_title: true
+The deploy folder referenced here would contain a folder 'test_task/task/' which contains Task.class and TaskLogic.class as well as a lib folder containing dependency jars.
 
 ### Python Lambda
 
-For non-Node.js Lambda code (e.g. python) uploaded as a .zip to an S3 bucket:
-
-```yaml
-  PyLambda:
-    handler: <file.py>.<function>               # eg:  lambda_handler.handler for lambda_handler.py with:  def handler(event, context):
-    timeout: <s>
-    s3Source:
-      bucket: '{{buckets.internal.name}}'       # refers to bucket set in config.yml
-      key: deploy/cumulus-process/<dir>/<file>  # eg: deploy/cumulus-process/modis/0.3.0b3.zip
-    runtime: python2.7                          # Node is default, otherwise specify.
-    layers:
-      - <some layer ARN>
-```
-
-## Other Lambda Configuration Options
-
-Other configurable options for Lambdas:
-
-```yaml
-  useXray: true             # Enable AWS X-Ray for the Lambda
-  launchInVpc: true         # Launch the Lambda in a VPC. Requires VPC configuration for the deployment.
-  logToElasticSearch: true  # Write Lambda execution logs to Elasticsearch.
-  useMessageAdapter: true   # Option to add/inject CMA as part of a cumulus workflow.  The CMA layer may be used instead
-  envs:                     # Add named environment variables for your Lambda.
-    - foo: 'bar'
-  layers:                   # Optional. e.g.: 'arn:aws:lambda:us-east-1:{{AWS_ACCOUNT_ID}}:layer:Cumulus_Message_Adapter:3'
-    - <layer1-arn>
-```
-
-To deploy all changes to `/tasks/` and `lambdas.yml`:
-
-```bash
-  $ kes cf deploy --kes-folder app --template node_modules/@cumulus/deployment/app --region <region> --deployment <deployment-name>
-```
-
-To deploy modifications to a single Lambda package:
-
-```bash
-  $ kes lambda <LambdaName> --kes-folder app --template node_modules/@cumulus/deployment/app --deployment <deployment-name>
-```
+Python Lambdas are created the same way as the Node.js example [above](#node.js-lambda).
 
-**Note:** By default, Cumulus workflows use versioned references to Lambdas and deploying a single Lambda does not update those references. So if you re-deploy just a single workflow Lambda, then any workflows using that Lambda will not be using the latest version of your Lambda code.
+## Cumulus Message Adapter
 
-You have to re-deploy your entire Cumulus application for workflows to reference the latest version of your Lambda code. Or you can disable workflow Lambda versioning by setting `useWorkflowLambdaVersions: false` for your deployment as a root key in your configuration.   For more information on this feature, see [the lambda version feature documentation](features/lambda_versioning.md).
+For Lambdas wishing to utilize the Cumulus Message Adapter(CMA), you should define a `layers` key on your Lambda resource with the CMA you wish to include. See the [input_output docs](workflows/input_output.md) for more on how to create/use the CMA.
```

Comment:
Can we link to the header about deploying the CMA?

Reply:
Good idea: 354986c2f